recipe/meta.yaml

```
@@ -226,7 +226,7 @@ outputs:
         - ml_dtypes >={{ ml_dtypes_version }},<0.5
         - numpy
         - opt_einsum >=2.3.2
-        - protobuf >=3.20.3,<5,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
+        - protobuf >=3.20.3,<6,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
```

I think we can simplify that condition. The way our `protobuf` is built (at least before v5) is that it depends on `libprotobuf`, which would conflict with the run_export that's being updated through the migration here, so nothing before v5 would be co-installable with the `tensorflow` being built here anyway.
```suggestion
        # simplified compared to upstream due to constraints arising through libprotobuf
        - protobuf >=5.26,<6
```